Bio/PDB/ResidueDepth.py

```diff
@@ -44,42 +48,422 @@
+        return _atomic_radii[8][typekey]
+    # Metals
+    elif at_elem == 'CU':
+        return _atomic_radii[20][typekey]
+    elif at_elem == 'ZN':
+        return _atomic_radii[19][typekey]
+    elif at_elem == 'MN':
+        return _atomic_radii[27][typekey]
+    elif at_elem == 'FE':
+        return _atomic_radii[25][typekey]
+    elif at_elem == 'MG':
+        return _atomic_radii[26][typekey]
+    elif at_elem == 'CO':
+        return _atomic_radii[28][typekey]
+    elif at_elem == 'SE':
+        return _atomic_radii[29][typekey]
+    elif at_elem == 'YB':
+        return _atomic_radii[31][typekey]
+    # Others
+    elif at_name == 'SEG':
+        return _atomic_radii[9][typekey]
+    elif at_name == 'OXT':
+        return _atomic_radii[3][typekey]
+    # Catch-alls
+    elif at_name.startswith(('OT', 'E')):
+        return _atomic_radii[3][typekey]
+    elif at_name.startswith('S'):
+        return _atomic_radii[13][typekey]
+    elif at_name.startswith('C'):
+        return _atomic_radii[7][typekey]
+    elif at_name.startswith('A'):
+        return _atomic_radii[11][typekey]
+    elif at_name.startswith('O'):
+        return _atomic_radii[1][typekey]
+    elif at_name.startswith(('N', 'R')):
+        return _atomic_radii[4][typekey]
+    elif at_name.startswith('K'):
+        return _atomic_radii[6][typekey]
+    elif at_name in set(('PA', 'PB', 'PC', 'PD')):
+        return _atomic_radii[13][typekey]
+    elif at_name.startswith('P'):
+        return _atomic_radii[13][typekey]
+    elif resname in set(('FAD', 'NAD', 'AMX', 'APU')) and at_name.startswith('O'):  # noqa: E501
+        return _atomic_radii[1][typekey]
+    elif resname in set(('FAD', 'NAD', 'AMX', 'APU')) and at_name.startswith('N'):  # noqa: E501
+        return _atomic_radii[4][typekey]
+    elif resname in set(('FAD', 'NAD', 'AMX', 'APU')) and at_name.startswith('C'):  # noqa: E501
+        return _atomic_radii[7][typekey]
+    elif resname in set(('FAD', 'NAD', 'AMX', 'APU')) and at_name.startswith('P'):  # noqa: E501
+        return _atomic_radii[13][typekey]
+    elif resname in set(('FAD', 'NAD', 'AMX', 'APU')) and at_name.startswith('H'):  # noqa: E501
+        return _atomic_radii[15][typekey]
+    else:
+        print('Warning: {}:{} not in radii library.'.format(at_name, resname))
+        return 0.01
+
+
 def _read_vertex_array(filename):
     """
     Read the vertex list into a Numeric array.
     """
     with open(filename, "r") as fp:
         vertex_list = []
-        for l in fp.readlines():
+        for l in fp:
             sl = l.split()
-            if not len(sl) == 9:
+            if len(sl) != 9:
                 # skip header
                 continue
             vl = [float(x) for x in sl[0:3]]
             vertex_list.append(vl)
     return numpy.array(vertex_list)
 
 
-def get_surface(pdb_file, PDB_TO_XYZR="pdb_to_xyzr", MSMS="msms"):
+def get_surface(model, MSMS="msms"):
```

Comment:
I suggest you leave ``PDB_TO_XYZR=...`` as the second argument, but as ``PDB_TO_XYZR=None``, and document this as being obsolete.

Then in the code, add something like:

```
if PDB_TO_XYZR is not None:
    # Raise warning or even an error about the API change.
```

That way it will work as is with existing scripts using the default.